geode-redis/src/test/java/org/apache/geode/redis/internal/SupportedCommandsJUnitTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.Test;
+
+public class SupportedCommandsJUnitTest {
+
+  private final String[] supportedCommands = new String[] {
```

Comment:
This seems painful but might be okay. I just don't like that we have to update two places (the enum and these lists) anytime we change what we support. But it probably will not happen that often.

Reply:
Definitely painful but at least it makes it a bit more deliberate when marking commands as un/supported. Should catch any inadvertent changes.